vayesta/solver/coupling.py

```diff
@@ -448,9 +446,122 @@ def _get_delta_t_for_extcorr(fragment, fock, solver, include_t3v=True, test_extc
                 assert(np.allclose(t3v_test, t3v))
 
     elif fragment.base.spinsym == 'unrestricted':
-        raise NotImplementedError
-        # TODO
-        pass
+        # Get ERIs and Fock matrix for the given fragment
+        (f_aa_ov, f_bb_ov), \
+                (v_aaaa_ooov, v_aabb_ooov, v_bbbb_ooov), \
+                (v_aaaa_ovov, v_aabb_ovov, v_bbbb_ovov), \
+                (v_aaaa_ovvv, v_aabb_ovvv, v_bbbb_ovvv) = _integrals_for_extcorr(fragment, fock)
```

Comment:
better to use a collection (dictionary?) or custom data structure here as the return value

Reply:
I think this would demand a larger change throughout vayesta to use a namespace or a dictionary instead of tuples, like I've done in `ebcc`.